test/extended/image_ecosystem/kubernetes_plugin.go

```diff
@@ -105,7 +105,7 @@ var _ = g.Describe("[image_ecosystem][jenkins] schedule jobs on pod slaves", fun
 			o.Expect(err).NotTo(o.HaveOccurred())
 
 			g.By("grant service account in jenkins container access to API")
-			err = oc.Run("policy").Args("add-role-to-user", "edit", "system:serviceaccount:"+oc.Namespace()+":jenkins", "-n", oc.Namespace()).Execute()
+			err = oc.Run("policy").Args("add-role-to-user", "edit", "system:serviceaccount:"+oc.Namespace()+":default", "-n", oc.Namespace()).Execute()
```

Comment:
default service account has as close to nothing as possible. by default, it can pull imagestream images in the namespace and that's it